types/react-native-fbsdk/index.d.ts

```diff
@@ -1,18 +1,20 @@
-// Type definitions for react-native-fbsdk 1.1
+// Type definitions for react-native-fbsdk 3.0
 // Project: https://github.com/facebook/react-native-fbsdk
 // Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
 //                 Thibault Malbranche <https://github.com/titozzz>
 //                 Stuart Forrest <https://github.com/stuartforrest-infinity>
+//                 Un Ha Kim <https://github.com/UHKim>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.1
```

Comment:
This should be set as low as possible. Did it need to change? It doesn’t look like it.

Reply:
Thank you for advice! @andrewbranch